examples/devices/__init__.py

```diff
@@ -0,0 +1,14 @@
+from dataclasses import dataclass
+
+from tickit.core.components.component import Component, ComponentConfig
+from tickit.core.components.device_simulation import DeviceSimulation
+
+from .counter import CounterDevice
+
+
+@dataclass
+class Counter(ComponentConfig):
+    """Simulation of simple counting device."""
+
+    def __call__(self) -> Component:  # noqa: D102
+        return DeviceSimulation(name=self.name, device=CounterDevice())
```

Comment:
This should be in `examples/devices/counter.py`

Reply:
according to the other examples, it has to be in the __init__.py in the same dir as the device

Reply:
Do all such examples happen to be in a directory with the name of the device?

Reply:
as in cryostream etc

Reply:
they all have their own sub dir in `tickit/devices` with an `__init__.py`

Reply:
So, by putting them in `__init__.py` you are effectively attaching them to the namespace of the parent directory, i.e. `tickit/devices/cryostream/Cryostream`, which would be equivalent to locating this at `tickit/examples/counter/Counter` by putting it in `tickit/examples/counter.py`

Reply:
Okay i've changed it